x-pack/legacy/plugins/lens/public/indexpattern_plugin/datapanel.tsx

```diff
@@ -298,6 +298,7 @@ export const InnerIndexPatternDataPanel = function InnerIndexPatternDataPanel({
                 'data-test-subj': 'indexPattern-switch-link',
                 className: 'lnsInnerIndexPatternDataPanel__triggerButton',
               }}
+              indexPatternId={currentIndexPatternId}
```

Comment:
Any tests for this?

Reply:
Both the `datapanel.test.tsx` and `layerpanel.test.tsx` test the behaviour of the inside of `ChangeIndexPattern`. The indexPatternId prop behaviour is already tested in `layerpanel.test.tsx` so I am not sure if the additional test is needed here 🤔